toolkit/tools/imagecustomizerschemacli/Makefile

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+BINARY_NAME := bin/imagecustomizerschemacli
+GO_FILES := $(wildcard *.go)
```

Comment:
Go already has good incremental builds. So, it is ok to just use a `.PHONY` rule and avoid this complexity.

Reply:
Remove `GO_FILES`?